server/build/webpack.js

```diff
@@ -356,7 +275,123 @@ export default async function createCompiler (dir, { buildId, dev = false, quiet
     },
     plugins,
     module: {
-      rules
+      rules: [
+        ...devLoaders,
+        {
```

Comment:
May be we can have other loaders defined above just like `devLoaders`

Reply:
👍